launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/servlets/CreateByLinkServlet.java

```diff
@@ -0,0 +1,147 @@
+package com.google.launchpod.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.launchpod.data.RSS;
+import com.google.launchpod.data.Channel;
+import com.google.common.base.Strings;
+
+@WebServlet("/create-by-link")
+public class CreateByLinkServlet extends HttpServlet {
+
+  private static final long serialVersionUID = 1L;
+  private static final String USER_FEED = "UserFeed";
+  private static final String EPISODE_TITLE = "episodeTitle";
+  private static final String EPISODE_DESCRIPTION = "episodeDescription";
+  private static final String EPISODE_LANGUAGE = "episodeLanguage";
+  private static final String MP3_LINK = "mp3Link";
+  private static final String BASE_URL = "https://launchpod-step18-2020.appspot.com/rss-feed?id=";
+  private static final String ID = "id";
```

Comment:
a lot of these are the same as our other sevlets. no need to fix now but they should be pulled into a constants file that all servlets reference to. 

reasoning is what if BASE_URL changes? we'll have to change it in all servlet files and it's very easy to forget one

Reply:
I was planning on making a PR for consolidating shared keys, methods, etc. after all of our different parts are merged!